cli/wallet/nep5.go

```diff
@@ -185,7 +185,7 @@ func getMatchingTokenAux(get func(i int) *wallet.Token, n int, name string) (*wa
 	var count int
 	for i := 0; i < n; i++ {
 		t := get(i)
-		if t != nil && (t.Name == name || t.Symbol == name || t.Address() == name || t.Hash.StringLE() == name) {
+		if t != nil && (t.Name == name || t.Symbol == name || t.Address() == name || t.Hash.StringBE() == name) {
```

Comment:
`name` comes from the user and user is always operating with LE hashes, so this change is wrong.